tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2882,18 +2764,17 @@ def test_tag_that_looks_like_aggregation(self):
         assert 1 == data["count"]
 
     def test_aggregate_negation(self):
-        project = self.create_project()
         data = load_data(
             "transaction",
-            timestamp=before_now(minutes=1),
-            start_timestamp=before_now(minutes=1, seconds=5),
+            timestamp=before_now(minutes=10),
+            start_timestamp=before_now(minutes=10, seconds=5),
         )
-        self.store_event(data, project_id=project.id)
+        self.store_event(data, project_id=self.project.id)
 
         features = {"organizations:discover-basic": True, "organizations:global-views": True}
         query = {
-            "field": ["event.type", "p99()"],
-            "query": "event.type:transaction p99():5s",
+            "field": ["event.type", "count()"],
+            "query": "event.type:transaction count():1",
```

Comment:
A condition where `p99():5s` can be really flakey because the `p99` is approximate so it might be slightly wrong. this is like doing equality checks with floating points. So I changed it to use `count():1` instead since it serves the same purpose to test negating aggregate conditions.